tabpfn_client/client.py

```diff
@@ -196,7 +196,10 @@ def try_connection(self) -> bool:
             if response.status_code == 200:
                 found_valid_connection = True
 
-        except httpx.ConnectError:
+        except httpx.ConnectError as e:
+            logger.error(f"Failed to connect to the server with error: {e}")
+            import traceback
```

Comment:
I'd move the import to the file for better readability :)

Reply:
ok i will do this in the new pr